src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -365,15 +365,17 @@ def _create_kubernetes(resource_group_name, deployment_name, dns_name_prefix, na
     smc = _resource_client_factory()
     return smc.deployments.create_or_update(resource_group_name, deployment_name, properties)
 
-def acs_get_credentials(dns_prefix, location):
+def acs_get_credentials(dns_prefix, location, output_file=None):
     # TODO: once we get the right swagger in here, update this to actually pull location and dns_prefix
     #acs_info = _get_acs_info(name, resource_group_name)
-    home = os.path.expanduser('~')
 
-    path = os.path.join(home, '.kube', 'config')
+    if not output_file:
+        home = os.path.expanduser('~')
+        output_file = os.path.join(home, '.kube', 'config')
```

Comment:
Suggestion: Make the parameter name --file / -f instead of --output-file. The prefix '--output' conflicts with the global parameter for specifying the output type.

When adding the `register_cli_argument(..)`, you can set the parameter name with `options_list=('--file', '-f')`.